style.css

```diff
@@ -303,3 +306,107 @@ ul {
   margin: 0;
   padding: 0;
 }
+
+.fotter-contact-form-container {
+  min-height: 150vh;
+  border-radius: 100px 0 0;
+  margin-top: 88px;
+  background-color: #6070ff;
+  overflow: hidden;
+  display: flex;
+}
```

Comment:
- [ ] Hello! I wanted to let you know that I noticed an issue in the "contact me" section that may be affecting the user interface. Specifically, there appears to be an unnecessary extra space at the bottom that creates an awkward gap and makes the section look unbalanced. Would you mind taking a look and removing the extra space? It would really help improve the overall look and feel of the page. Thank you for your attention to this matter!
- Here's how it looks:
  ![image](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/104968236/fef830fb-f91b-4059-8146-ca320e5ad3cb)